Source/Events.Processing/Streams/Partitioned/FailingPartitions.cs

```diff
@@ -172,7 +179,8 @@ async Task<(StreamProcessorState, FailingPartitionState)> SetFailingPartitionSta
         Task<IProcessingResult> RetryProcessingEvent(FailingPartitionState failingPartitionState, CommittedEvent @event, PartitionId partition, CancellationToken cancellationToken) =>
             _eventProcessor.Process(@event, partition, failingPartitionState.Reason, failingPartitionState.ProcessingAttempts - 1, cancellationToken);
 
-        Task PersistNewState(IStreamProcessorId streamProcessorId, StreamProcessorState newState) => _streamProcessorStates.Persist(streamProcessorId, newState, CancellationToken.None);
+        Task PersistNewState(IStreamProcessorId streamProcessorId, StreamProcessorState newState) =>
+            _streamProcessorStates.Persist(streamProcessorId, newState, CancellationToken.None);
```

Comment:
This should take in a `CancellationToken` from the outside to use. Hopefully it is not used, but if we can't cancel this call, we could end up in a situation where a StreamProcessor is never stopped and therefore cannot be restarted.

Reply:
https://github.com/dolittle-runtime/Runtime/pull/408/commits/619bfca5d0315b408368e016311452e94e7f6b1b